### PR TITLE
Updated install script

### PR DESCRIPTION
**Pull request by BlakePerrin13**
I updated the install script for setting up the Pi as an Access Point/Router. I also moved the README and setup.sh script to a better location so it is easier to access. The install script also has a few checks to make sure that the required directories are created/installed before editing any configurations. I also changed a few of the filenames and directory names so they would work correctly with the script.

### CODE REVIEW

**Review by J4keFr0mStatef — APPROVED** [review]
Looks good.  I like the changes made to the readme and the install script.  Good on you remembering to change CSNA to Phoebus.

File: APSetup/Files/etc/hostapd/hostapd.conf [hunk 1]
```diff
@@ -16,7 +16,7 @@ rsn_pairwise=CCMP
 # Configuration for an AP on wlan1 (5G)
 interface=wlan1
 driver=nl80211
-ssid=CSNA
+ssid=Phoebus
```
Comment (CamJak) [inline APSetup/Files/etc/hostapd/hostapd.conf:19]: This is extremely minor but should Pheobus be caps?

File: APSetup/README.md
Comment (CamJak) [inline APSetup/README.md:1]: Moved to a much better location! Should be easier to find and use now

File: APSetup/Files/etc/ssh/phoebus_banner
Comment (CamJak) [inline APSetup/Files/etc/ssh/phoebus_banner:1]: Did this actually change, or are you waiting to change the banner from CSNA?

File: APSetup/Files/etc/hostapd/hostapd.conf (hunk 1 above)
Comment (BlakePerrin13) [reply APSetup/Files/etc/hostapd/hostapd.conf:19]: I dont think it matters. Its just based on what you want the SSID to look like when its broadcasted

Comment (CamJak) [reply APSetup/Files/etc/hostapd/hostapd.conf:19]: Gotcha, this'll be changeable by the user anyways

File: APSetup/Files/etc/ssh/phoebus_banner
Comment (BlakePerrin13) [reply APSetup/Files/etc/ssh/phoebus_banner:1]: I only changed the filename. I figured we could just change the art to whatever we wanted earlier

Comment (CamJak) [reply APSetup/Files/etc/ssh/phoebus_banner:1]: Gotcha, just leave it as whatever for now and change it later?

Comment (BlakePerrin13) [reply APSetup/Files/etc/ssh/phoebus_banner:1]: Yeah that was my plan

**Review by CamJak — APPROVED** [review]

**Pull request merged**